File: satusehat-integration-sdk/sdks/python/examples/basic.py
import json
from pathlib import Path
from satusehat_sdk import SatusehatConfig, SatusehatSdk
cfg=SatusehatConfig.from_env()
sdk=SatusehatSdk(cfg)
sample = Path(__file__).resolve().parents[3] / 'examples' / 'fhir' / 'encounter.json'
with sample.open('r', encoding='utf-8') as f:
    payload=json.load(f)
eid=sdk.enqueue('POST','Encounter',payload=payload)
print('queued',eid)
print('processed',sdk.process_once(20))
sdk.close()

File: sdks/node/examples/basic.mjs
import {readFileSync} from 'node:fs';
import {SatusehatConfig,SatusehatSdk} from '../src/index.js';
const sdk=new SatusehatSdk(SatusehatConfig.fromEnv());
const encounter=JSON.parse(readFileSync(new URL('../../../examples/fhir/encounter.json', import.meta.url),'utf8'));
const id=sdk.enqueue('POST','Encounter',undefined,encounter);
console.log('queued',id);
console.log('processed',await sdk.processOnce(20));
sdk.close();
